src/book/03-guides/03-hardware/01-working-with-robots.mdx

```diff
@@ -37,63 +31,44 @@ To align an ankle correctly perform the following instructions:
-      the same direction the horn was orientated. The shoulder should look like it is on backwards.
-      ![shoulder orientation](./images/shoulder/shoulder_orientation2.jpg 'shoulder orientation')
-    - Feed the chord through the gap in the shoulder and connect the arm again to the slot between the shoulder and head
-    - slide the arm in place and rescrew the arm to the shoulder
-
-    Note: If you don't place the cords of the arm far enough into the shoulder, they will get caught and will limit the shoulder movements. If this happens, do not force the arm to move, but take apart the shoulder and redo the attatchment.
-
-7.  Testing
-
-    - # Now the shoulder needs to be tested using Open Dynamixel Wizard. This tests if the soulder was placed on correctly, or the servo has come out of alignment during the attatchment of the shoulder and arm.
-          - Get the new servo set with the same serial number on the box
-          - Connect the metal circular disk that came in the box on the new servo, ensuring you line up the indents on the metal disk and servo
-          - Place the servo into the body, put back the plastic holder and reconnect all the chords that were disconnected in the 'removing servo' step
-          - Rescrew the four screws that were taken out
-          - The arms of the robot should be directly outwards when set to 0 degrees. This means you need to orientate the server before placing the shoulder back on.
-          On the metal circular disk that was placed on, there are small indents in the metal, two on one side and one on the other.
-          The single indent needs to be towards the front of the robot, and the two dots towards its back, this ensures the robots shoulder will be correctly aligned.
-
-8.  Reconstruction
-
-    - Now attach the shoulder back on, ensuring you put back the plastic spacers and leave one hole without a screw (the biggest hole in the circle of screws). The shoulder should also be placed on
-      with correct orientation. On the shoulder there is one side of raised up plastic, two sides with the holes for the screws and a shorter side. The side with the shortest bit of plastic should be faced frontwards,
-      the same direction the metal circular disk was orientated.
-    - Connect the chord of the arm again to the slot between the shoulder and head
-    - slide the arm in place and rescrew the arm to the shoulder
-
-9.  Testing - Now the shoulder needs to be calibrated, see servo calibration for this.
-    > > > > > > > 8f25ae92838312b8020ba3e2a14bd2ceeaf324c5
+- On the shoulder of the robot, there is a ring of screws front and back. Remove all the screws and the larger centre screw on both sides
+- Remove the arm from the shoulder socket, it will still be attatched by a wire
```

Comment:
```suggestion
- Remove the arm from the shoulder socket by carefully stretching the shoulder cuff around the two larger centre screws, it will still be attatched by a wire
```